recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from conan import ConanFile
+from conans import CMake, tools
```

Comment:
```suggestion
from conan.tools.cmake import CMake
```
Please, avoid mixing `conans` and `conan`. With Conan 2.x, `conans` will be deprecated, so we are migrating everything.

Also, consider CMakeToolchain generator, much better that only workaround with `conan_basic_setup()` injected in your cmake file.


Consider https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/cmake_package  as example